### ISSUE TITLE
Pdf creation error

### ISSUE THREAD

**Issue by jkovesi** [stolksdorf/homebrewery#227]
Hi,

I have a problem with the following doc. When I try to create pdf it won't be ok. some of tables are inconsistent. For example the goblins table. In editor mode is everything is ok. 
I already checked all div-s and corrected lot of problem, but it is still wrong. 
Could you check it?

BR,

Jozsef Kovesi

ps.: Thanks for this tool it is great help to me! :)

### Additional Details

**Share Link** : http://homebrewery.naturalcrit.com/edit/HJgbwrvEp

or

**Brew code to reproduce** : <details><summary>Click to expand</summary><code><pre>

PASTE BREW CODE HERE

</pre></code></details>

**Comment by jkovesi** [comment]
Here is an example: http://imgur.com/a/RgVxd

**Comment by stolksdorf** [comment]
No idea what's causing this. Seems like an issue with Chrome's PDF renderering, which is incredibly difficult to debug

**Comment by infrasparker** [comment]
Having the same issue, but not just with Chrome. I've encountered the same/similar issues using Firefox and Microsoft Edge. I looked at other issue threads that mentioned switching to Chrome 54(beta) instead of 53, but I've used both versions 54 and 55(beta) all with the same result. Looks fine on both the editor page and the share page, but when it opens up to print/save pdf on any browser, almost all the tables get distorted.
Share Link: http://homebrewery.naturalcrit.com/share/Bk4EXy_bl
